lib/Clownface.js

```diff
@@ -94,18 +138,39 @@ class Clownface {
     }
   }
 
+  /**
+   * Returns an array of graph pointers where each on has a single _context
+   *
+   * @returns {Clownface[]}
+   */
   toArray () {
     return this._context.map(context => Clownface.fromContext(context))
   }
 
+  /**
+   * Returns graph pointers which meet the condition specified in a callback function
+   * @param {FilterCallback} callback
+   * @returns {Clownface}
+   */
   filter (callback) {
     return Clownface.fromContext(this._context.filter(context => callback(Clownface.fromContext(context))))
   }
 
+  /**
+   * Performs the specified action on every graph pointer
+   * @param {ForEachCallback} callback
+   * @returns {Clownface}
+   */
   forEach (callback) {
     return this.toArray().forEach(callback)
   }
 
+  /**
+   * Calls a defined callback function on each graph pointer, and returns an array that contains the results.
```

Comment:
Returns a new array with the result of applying `callback` on each graph pointer?